migrations/1683712628_tagList.ts

```diff
@@ -0,0 +1,64 @@
+import { Client, SimpleSchemaTypes } from "@datocms/cli/lib/cma-client-node";
+
+export default async function (client: Client) {
+  const newFields: Record<string, SimpleSchemaTypes.Field> = {};
+  const newItemTypes: Record<string, SimpleSchemaTypes.ItemType> = {};
+
+  console.log("Create new models/block models");
+
+  console.log('Create block model "Structured text - Structured text - Tag list" (`tag_list`)');
```

Comment:
`Structured text` is written twice, and `tag_list` should be `structured_text_tag_list`